README.md

```diff
@@ -188,7 +188,28 @@ module.exports = React.createClass({
 })
 ```
 
-### How to write your own loaders/wrappers
+### How to use your own loaders
+
+Create a `gatsby.config.js` in the root of your
+project. `gatsby.config.js` exports a function which accepts a config
+object and an environment string. The environment string will be one
+of `serve`, `static` or `production`.
```

Comment:
Should be develop now. Plus we should point them to the default loaders somehow so they know what they're modifying.